.github/workflows/tests.yml

```diff
@@ -27,9 +28,8 @@ jobs:
             os: ubuntu-20.04
           - py: "3.5"
             os: macos-12
-          - py: "3.4"
-            os: windows-2019
-            installer: easy_install
+    env:
+      PIP_TRUSTED_HOST: ${{ matrix.py == '3.5' && 'pypi.python.org pypi.org files.pythonhosted.org' || '' }}
```

Comment:
Could you add a comment to explain why we need to do this for Python 3.5? (I see we did it unconditionally before; I assume it's not needed for more-recent versions of Python?).

Reply:
Sure, but fwiw I'm going to be following up with another change right after this to bump min supported py version and remove the workaround again. This was just the easiest way to break up small self-contained changes for testing vs bigger stuff. 

The short version is just that pip is newer than legacy setuptools/easy_install and really really old systems seem to not have it working correctly.

Reply:
Oh sorry, misread this and thought you were asking why easy_install was conditionally needed. Added a comment for trusted hosts. And I saw there may be other better workarounds when I checked that link again, but also for just a test runner action I feel like the stakes are fairly low for that kind of security risk.